pages/1.13/installing/production/system-requirements/index.md

```diff
@@ -65,9 +65,12 @@ There are many mixed workloads on the masters. Examples of mixed workloads on th
   | _/var/lib/dcos/cockroach_ | CockroachDB [enterprise type="inline" size="small" /] |
   | _/var/lib/dcos/navstar_ | for Mnesia database |
   | _/var/lib/dcos/secrets_ | secrets vault [enterprise type="inline" size="small" /] |
+  | _/var/lib/dcos/exec_ | Temporarily needed files from various DC/OS services. |
```

Comment:
The content seems all right. Is there any reason why you have moved `noexec` note under the table? For example the `_/var/lib/dcos/exhibitor/zookeeper/transactions_` has advisory in the table itself.

Reply:
I'm OK to move it if you prefer. In my mind the later section could be used to explain how to clean up the directory when we get to that, which could grow. Let me know if you'd like that moved here.